Community/General/Sources/ML/MLXMarkerListCommunityModules/XMarkerListFile/mlXMarkerListToFile.cpp

```diff
@@ -2,8 +2,8 @@
 //! The ML module class XMarkerListToFile.
 /*!
 // \file    mlXMarkerListToFile.cpp
-// \author  Coert Metz
-// \date    2007-07-06
+// \author  Coert Metz, Erwin Vast
+// \date    2007-07-06, 20-11-15
```

Comment:
Don't you prefer the ISO date format (unambiguous for all days/months, alphanumerically sortable, ...)?